src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -285,6 +291,18 @@
       text: az spring-cloud app identity show -n MyApp -s MyCluster -g MyResourceGroup
 """
 
+helps['spring-cloud app identity force-set'] = """
+    type: command
+    short-summary: Force set managed identities on an app.
+    examples:
+    - name: Force remove all managed identities on an app.
+      text: az spring-cloud app identity force-set -n MyApp -s MyCluster -g MyResourceGroup --system-assigned disable --user-assigned disable
+    - name: Force remove all user-assigned managed identities on an app, and enable or keep system-assigned managed identity.
+      text: az spring-cloud app identity force-set -n MyApp -s MyCluster -g MyResourceGroup --system-assigned enable --user-assigned disable
+    - name: Force remove system-assigned managed identity on an app, and assign or keep user-assigned managed identities.
+      text: az spring-cloud app identity force-set -n MyApp -s MyCluster -g MyResourceGroup --system-assigned disable --user-assigned IdentityResourceId1 IdentityResourceId2
+"""

```

Comment:
Why do we need this command? Can its function be replaced by the `spring-cloud app identity remove` command?

Reply:
Normally we don't need these commands. It's related to a data consistency problem among managed identity RP and my RP. There can be case when user identity is assigned to an app in my side, but deleted in managed identity RP due to data in-consistency.  These commands leverage put method to reset data back with one command.

app identity remove/assign is done with patch http method, there can be case, there is no identity assigned to an app in my RP, but managed identity RP still has some identity assigned to the app.

Reply:
Got it, thanks~